docs/docs/integrate/custom-apis/_category_.yml

```diff
@@ -0,0 +1,4 @@
+position: 14
```

Comment:
should this section go under `build` now as previously it was under `develop` ?

Reply:
The original thinking behind the refactored docs structure is an end-to-end user journey, where a new person that discovers Rill could easily follow our docs to get an initial project deployed and set up (on Rill Cloud). If you're curious, you can check out this [Notion doc](https://www.notion.so/rilldata/Rill-Docs-Structure-52de7fe35d5a4e598ecf835ccaf4807d).

Logically speaking, we should be putting this section under either `build` or `integrate`. I would consider custom APIs more of an "advanced" concept and not really necessary for most end users so I personally think it makes the most sense to keep it under `integrate` (which is really more for developers). However, I can see the argument for `build` as well as it's still part of the building process, just for more advanced project setups. @mindspank Did you want to make a judgment call on where these docs should live in our new structure?

Reply:
Ok makes sense, feel free to merge  either way you decide. I am ok with both.

Reply:
Feels like integrate is the correct place